plugin/sampling/strategystore/static/strategy_store_test.go

```diff
@@ -293,6 +346,49 @@ func TestAutoUpdateStrategy(t *testing.T) {
 		time.Sleep(1 * time.Millisecond)
 	}
 	assert.EqualValues(t, makeResponse(sampling.SamplingStrategyType_PROBABILISTIC, 0.9), *s)
+
+	// Test auto update strategy with URL option.
```

Comment:
This seems like a pretty separate testing scenario, any reason why it cannot be moved into a new test function with appropriate name?

Reply:
Yes, it makes sense to move it to a separate test function.